### ISSUE TITLE
Xcode 8.3 stuck at compiling GoogleMaterialIconFont

### ISSUE THREAD

**Issue by ArturGasparyan** [kitasuke/GoogleMaterialIconFont#26]
Hello,

today I updated to Xcode 8.3. While building my app (which worked!) the compilation is stuck at "Building GoogleMaterialIconFont" (see attached image).

Any ideas what goes wrong? How can this be fixed?

Kind regards
![googlematerialiconfont](https://cloud.githubusercontent.com/assets/1446843/24586824/6d620b76-17a9-11e7-9106-8e620737ff9a.png)

**Comment by n1mda** [comment]
I have the same problem, using swift3.0 branch

**Comment by ArturGasparyan** [comment]
Yes, I'm also on the swift3.0 branch. Kindly find time to fix this, please.

**Comment by se-rg-os** [comment]
I am facing the same issue even after updating project to Swift 3.0 (from separate branch and my own changes). 

It works though in Xcode 8.2.1

But in Xcode 8.3 I am getting an error that "Module compiled with Swift 3.0.2 cannot be imported in Swift 3.1 ..." - fighting with this one for now.

**Comment by ArturGasparyan** [comment]
Any updates here, @kitasuke? Please help

**Comment by n1mda** [comment]
I switched over to `SwiftIconFont` for this reason

**Comment by fabiangiger** [comment]
@n1mda Thanks... SwiftIconFont is a good alternative.
